app/controllers/user_controller.rb

```diff
@@ -0,0 +1,24 @@
+class UserController < ApplicationController
```

Comment:
Not sure if this was an error before the first merge but I don't think we need two user controllers.

Reply:
good catch, i'll delete this